### DISCUSSION TITLE
Getting Started

### DISCUSSION THREAD

**Discussion by stefangpeter** [Homebrew/discussions#4757] [Getting started]
### Output of `brew config`

```shell
?
```

### Output of `brew doctor`

```shell
?
```

### Description of issue

Hello, I don't know if this is the right place for my problem, Feel free to educate me. ;)

Problem:
We use circleci to build the native parts of our software. There is one build done for the mac clients. Xcode 14.3.1 etc. The result worked fine up until end of august. Suddenly the results from the build did not start anymore on the mac os.
After hours of forensic what has changed between the last working build (24.8.23) and the day it didn't anymore (29.8.) A friend and I finally found that the bottle for qt5 has changed in name between the builds. Currently the bottle 5.15.10_1.ventura.bottle.tar.gz is now used (mirror: https://mirrors.tuna.tsinghua.edu.cn/linuxbrew-bottles/bottles/?C=M&O=D) Something must have changed from the former (not to be found anymore: 5.15.10.ventura.bottle.1.tar.gz). In the older package the resulting App Bundle was containing some *.dylibs in the framework folder (libpng... etc) which where needed by QtGui and QtCore for example. But in the new package this libs are not deployed to the bundle anymore but looked for at run time on the underlying system?!"?!? I dont know why.

No my questions are:
- how can I get the old working bottle back?
- where can I see who has made the change in the bottle?
- why was the change made?
- how can convince the buildprocess to add needed subdependencies to an app bundle? which seemed to have worked before

Thanks for your help.

**Comment by SMillerDev** [comment]
> how can I get the old working bottle back?

You can't, you'll have to build the earlier version for yourself.

> where can I see who has made the change in the bottle?

https://github.com/Homebrew/homebrew-core/commits/master

> why was the change made?

See the attached pull requests

> how can convince the buildprocess to add needed subdependencies to an app bundle? which seemed to have worked before

That depends on your build process I guess.

**Reply by stefangpeter** [reply]
thanks for the fast answer.

it seems the commit was 9b99c9a256280d85a33ab7d8118e59c0ff0a3b6e
it was made from the BrewTestBot? What is it, what does it do?
And there is no way I can find out why it has been done or what has changed in the bottle?
And the build process uses just brew install qt@5, builds the product and calls macdeployqt to create the app bundle. which is now missing some needed dependencies. :(

**Reply by ZhongRuoyu** [reply]
This looks like https://github.com/Homebrew/homebrew-core/issues/140930, which is being looked into.

**Reply by stefangpeter** [reply]
thanks for directing me to this. This seems indeed to be the exact same problem.

**Comment by stefangpeter** [comment]
thx, it has been fixed over the weekend.
